server/services/interventionsService.test.ts

```diff
@@ -1797,45 +1797,6 @@ pactWith({ consumer: 'Interventions UI', provider: 'Interventions Service' }, pr
     })
   })
 
-  describe('createActionPlanAppointment', () => {
```

Comment:
Ah, I didn't realise you'd done this as part of this PR, so I've done it in a separate commit in another one. Shall we merge that one https://github.com/ministryofjustice/hmpps-interventions-ui/pull/216 and then rebase this one?